tests/ts/lsfd/mkfds-unix-in-netns

```diff
@@ -68,8 +69,11 @@ compare_net_namespaces()
 	    compare_net_namespaces "$t" "${PID}"
 
 	    kill -CONT "${PID}"
-	    wait "${MKFDS_PID}"
 	fi
+	wait "${MKFDS_PID}"
+	if [[ $? == "$UNSUPPORTED" ]]; then
+	    ts_skip "unshare(2) is not permitted"
```

Comment:
permitted or unsupported? The "permitted" sounds like the user who executes the test have no permissions.

Reply:
BTW, for pidfs_open(2), there is "not available". (sorry for nitpicking:-)

Reply:
As I wrote in https://github.com/util-linux/util-linux/pull/1845#issuecomment-1324621435 I will withdraw (or revise) these changes.